scripts/tokenization_document.py

```diff
@@ -0,0 +1,102 @@
+#
```

Comment:
script names should begin with verbs - i.e., they're actions.

how about `tokenize_corpus`?